examples/files/src/main/java/com/hazelcast/jet/examples/files/SalesJsonAnalyzer.java

```diff
@@ -41,7 +42,7 @@ private static Pipeline buildPipeline(String sourceDir) {
 
         BatchSource<SalesRecord> source = Sources.filesBuilder(sourceDir)
                                                  .glob("*.json")
-                                                 .buildJson(SalesRecord.class);
+                                                 .build(JsonUtil.asJson(SalesRecord.class));
```

Comment:
this should read a json which has a line per sales record, right now it's a list